Packs/Zerohack_XDR/Integrations/ZerohackXDR/README.md

```diff
@@ -0,0 +1,2 @@
+Note: This is a beta pack, which lets you implement and test pre-release software. Since the pack is beta, it might contain bugs. Updates to the pack during the beta phase might include non-backward compatible features. We appreciate your feedback on the quality and usability of the pack to help us identify issues, fix them, and continually improve.
+
```

Comment:
Please create a proper README file for the integration. You can generate on easily using our SDK command generate-docs https://github.com/demisto/demisto-sdk/blob/master/demisto_sdk/commands/generate_docs/README.md